tests/setTest.js

```diff
@@ -76,4 +76,52 @@ describe('Set', () => {
     })
   })
 
+  context('union()', () => {
+    it('unions the set with another set and returns the resulting set.', () => {
+      const mySet = new Set([1, 2, 3, 4])
+      const newSet = new Set([3, 4, 5, 6, 7])
+      const unionSet = mySet.union(newSet)
+      expect(unionSet.set).to.eql([1, 2, 3, 4, 5, 6, 7])
```

Comment:
It's up to you, but my preference is for `to.deep.equal`, because it makes the fact that you're doing a deep equality comparison more specific (instead of just removing vowels from `eql` - no idea why `Chai` decided to do that).